protocol/src/test/kotlin/models/MessageTest.kt

```diff
@@ -0,0 +1,35 @@
+package protocol.models
```

Comment:
Within our tests we have packages...

- `models`
- `protocol`
- `protocol.models` 

We should reconcile

Reply:
Didn't you have a PR open to fix that?